src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt

```diff
@@ -218,9 +218,9 @@ class TakenRESTService @Inject constructor(
             taakdata[TaakVariabelenService.TAAK_DATA_DOCUMENTEN_VERZENDEN_POST]?.let {
                 updateVerzenddatumEnkelvoudigInformatieObjecten(
                     documenten = it,
-                    // implicitly assume that the following keys are present in taakdata
+                    // implicitly assume that the verzenddatum key is present in taakdata
                     verzenddatumString = taakdata[TaakVariabelenService.TAAK_DATA_VERZENDDATUM]!!,
-                    toelichting = taakdata[TaakVariabelenService.TAAK_DATA_TOELICHTING]!!
+                    toelichting = taakdata[TaakVariabelenService.TAAK_DATA_TOELICHTING]
```

Comment:
Is there no default reason that should be used if none is given?

Reply:
@camielb @edgarvonk Do we have a default reason for completing a task? If not should we have one? WDYT?

Reply:
@hsiliev I have no idea. I think that is something to check with our PO?